Formula/octant.rb

```diff
@@ -38,7 +38,7 @@ def install
       system "go", "generate", "./pkg/plugin/plugin.go"
       system "go", "run", "build.go", "web-build"
 
-      build_time = Utils.safe_popen_read("date -u +'%Y-%m-%dT%H:%M:%SZ' 2> /dev/null").chomp
+      build_time = Time.now.strftime("%Y-%m-%dT%H:%M:%S%Z")
```

Comment:
Same here:

```suggestion
      build_time = Time.now.utc.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Reply:
Yup, this was intentional. The trailing `Z` is meaningless; it's almost certainly a typo for `%Z`.